app/src/main/java/org/ometa/lovemonster/ui/adapters/LoveArrayAdapter.java

```diff
@@ -0,0 +1,101 @@
+package org.ometa.lovemonster.ui.adapters;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import org.ometa.lovemonster.R;
+import org.ometa.lovemonster.models.Love;
+import org.ometa.lovemonster.models.User;
+
+import java.util.List;
+
+/**
+ * Created by bschmeckpeper on 11/18/15.
+ */
+public class LoveArrayAdapter extends ArrayAdapter<Love> {
+
+
+    class VisibilityToggler implements View.OnClickListener {
```

Comment:
this class definition should be static, since it doesn't need a reference to the enclosing instance.

some background:
http://programmers.stackexchange.com/questions/238782/why-prefer-non-static-inner-classes-over-static-ones